### ISSUE TITLE
Wrong password on Firefox and chrome for iOS 

### ISSUE THREAD

**Issue by ralyodio** [bitwarden/mobile#1876]
### Steps To Reproduce

I use Bitwarden on iOS. Seems to work fine with safari. But for Firefox and chrome I’m getting an old password (I guess). If I copy password directly from Bitwarden app it works fine on chrome and Firefox but not when integrated. 

Site is Grazily.com. I don’t think it’s a ui but on my site. 

### Expected Result

Password should be correct. 

### Actual Result

Wrong password. Can’t login 

### Operating System

iOS

### Operating System Version

Latest

### Device

iPhone 12 Pro 

### Build Version

15.3.1

### Beta

- [ ] Using a pre-release version of the application.

**Comment by yashwalter12** [comment]
Hi @ralyodio , We tried to reproduce the issue with the steps mentioned above and in the Firefox and chrome on iOS. Each time I am able to login with with latest update password(if it is correct), if it is not correct password not able to login. Could you please give little more information on
1. Is this still happening to you
2. Did you update the password after first successful login?
3. Did you update the password on the Bitwarden manually or thru autofill?

**Comment by ralyodio** [comment]
yes its still happening. If i copy the password from bitwarden app and paste into login form I can login fine. If I try to use bitward autofill it says "password is incorrect". I guess its a caching issue or something no idea.

Safari works fine.

**Comment by yashwalter12** [comment]
Hello @ralyodio  Thank you for the input. Can you specify what is your build version you are using for the app?

**Comment by ddurst** [comment]
I can reproduce this on iOS 15.7.1 on an iPhone 7, on Firefox and Chrome but not Safari, using Version: 2022.10.0 (2347).

I have iOS set to autofill using bitwarden, and if I just select the login to use, it fails (the site I am trying this on is anthem.com) — but if I edit the password (by erasing the last character and retyping it), it works.

**Comment by ddurst** [comment]
So to be clear: it is filling the correct password. But the password doesn’t work unless I manually type a part of it. (If that makes sense.)

**Comment by vvolkgang** [comment]
Issue migrated to https://github.com/bitwarden/mobile/issues/1876
